test/models/User.test.js

```diff
@@ -0,0 +1,91 @@
+require('dotenv').config();
+require('../../lib/utils/connect')();
+const mongoose = require('mongoose');
+const { Types } = require('mongoose');
+const User = require('../../lib/models/User'); 
+const { tokenize } = require('../../lib/utils/token');
+
+describe('models', () => {
+  beforeEach((done) => { 
+    mongoose.connection.dropDatabase(done);
+  });
```

Comment:
```suggestion
  });
  afterAll((done) => {
      mongoose.connection.close(done);
  });
```
An afterAll statement is required to close mongoose connection.